src/Symfony/Bridge/Doctrine/Tests/Validator/DoctrineLoaderTest.php

```diff
@@ -108,6 +108,16 @@ public function testLoadClassMetadata()
         $this->assertCount(1, $embeddedMaxLengthConstraints);
         $this->assertInstanceOf(Length::class, $embeddedMaxLengthConstraints[0]);
         $this->assertSame(25, $embeddedMaxLengthConstraints[0]->max);
+
+        $this->assertEmpty($classMetadata->getPropertyMetadata('guidField'));
```

Comment:
if you change this to `$this->assertCount(0, $classMetadata->getPropertyMetadata('guidField'));` (same on the following line), your fix looks perfect to me

Reply:
Done.